README.md

```diff
@@ -1,27 +1,78 @@
-# goitneo-python-hw-3-group-3
+# CLI Personal Assistant
 
 ## Description
 A CLI organizer that can help store contacts' phone numbers and birthday dates.
 
-
 ## Get Started
 
-Run `pip install personal_assistant` to install the package and then `get-started` to get started. The list of available commands is as follows:
+Run `pip install personal_assistant` to install the package and then `get-started` to get started.<br><br>
+
+**The list of available commands is as follows:**
 
 - `hi` or `hello` or `greeting` - to say hi to your favorite PA!
+- `contacts` - to enter the address book interface;
+- `notes` - to enter the notes interface;
+- `close` or `exit` - to exit the personal assistant;
+
+## Interface Address book
+
 - `add name phone` - to add a acontact and its phone number;
+- `delete name` or `remove name` - to delete the contact;
+- `find` - to show one contact from the list;
+- `entries` - to show the number of contacts;
 - `all` - to list all the contacts and their phone numbers;
-- `phone name` - to list all the phone numbers associated with the contact; 
-- `change name old phone new phone` - to change the phone number;
-- `remove-phone name phone` - to remove the phone number;
+- `help` - to show the list of available commands for the address book;
+- `back` - to exit the address book;
+
+### Phone
+- `show-phone name` - to list all the phone numbers associated with the contact; 
+- `change-phone name old phone new phone` - to change the phone number;
+- `delete-phone name phone` - to remove the phone number;
+
+### Birthday
 - `add-birthday name DD.MM.YYYY` - to add birthday;
 - `show-birthday name` - to show birthday associate with the contact;
+- `change-birthday name` - to change the contact's date of birth;
+- `delete-birthday name` - to remove the contact's date of birth;
 - `birthdays` - to show the list of contacts who have a birthday within the following 7 days;
-- `delete name` or `remove name` - to delete the contact;
-- `entries` - to show the number of contacts
-- `close` or `exit` - to exit the personal assistant;
 
-## Notes:
+### Email
+- `add-email name email` - to add email;
+- `show-email name` - to show email associate with the contact;
+- `change-email name old email new email` - to change the email;
+- `delete-email name` - to remove the email;
+
+### Address
+- `add-address name address` - to add address;
+- `show-address name` - to show address associate with the contact;
+- `change-address name old address new address` - to change the address;
+- `delete-address name` - to remove the address;
+
+## Interface Notes
+
+- `add text` - to create a note;
+- `show note number` - to show one note;
+- `change note number new text` - to change one note;
```

Comment:
the current command is `edit-note` + additional input (name), but don't change it yet